controllers/scripts/update-pod-status.sh

```diff
@@ -73,7 +73,7 @@ else
     echo "Pod restarted"
 fi
 
-echo $AERO_CLUSTER_JSON | python3 create_pod_status_patch.py $MY_POD_NAME
```

Comment:
The cluster JSON could be larger than command line character limits.
Instead, write the json to a file and pass the filename to the patch script.

Reply:
I'll include a request for json inside the script it will eliminate the problem

Reply:
Or you could store the JSON in a file from the curl directly and the pass the filename to the python script to eliminate the bash variable AERO_CLUSTER_JSON altogether.